wom/src/test/scala/wom/util/YamlUtilsSpec.scala

```diff
@@ -74,12 +74,6 @@ class YamlUtilsSpec extends FlatSpec with Matchers with TableDrivenPropertyCheck
       refineMV[NonNegative](0),
       null,
     ),
-    (
-      "an empty yaml",
-      "",
-      refineMV[NonNegative](1),
-      "null",
```

Comment:
Was this removed because it was inconvenient, or genuinely found to be wrong?

Reply:
There's a `legalYamlTests` down below that it should be moved to if it is now a valid input.

Reply:
I'll move it